Integrations/ArcSightESMv2/ArcSightESMv2.py

```diff
@@ -549,10 +582,29 @@ def update_case(case_id, stage, severity):
     return_error('Failed to update case, fail to parse response. Response Body: {}'.format(res.text))
 
 
+@logger
+def get_correlated_events_ids(event_ids):
+    related_ids = set(event_ids)
+    correlated_events = beautifully_json(get_security_events(event_ids, ignore_empty=True))
+
+    for raw_event in correlated_events:
+        if raw_event.get('baseEventIds'):
+            try:
+                related_ids.add(raw_event.get('baseEventIds'))
+            except Exception as ex:
```

Comment:
@orenzohar lets have a short talk about this one.
I don't understand what exception do you expect here, and in any case this seems too general and/or can be replaced with an if statement.

Reply:
if the object we want to add to the set through `add()` is not hashable (e.g., list) it will throw an Exception, and if it is not iterable (e.g., int) `update()` will throw an Exception.

Reply:
a way to simplify this would be to use only a list all along, and return `list(set(the_list))`

Reply:
or something like this maybe?
![image](https://user-images.githubusercontent.com/44546251/54364873-7389b280-4676-11e9-87b4-c6ff9edc7ed1.png)

Reply:
yes this last example from pycharm is much better in 2 ways:
1. it dose not create the risk of silencing an exception we did not expect.
2. the expected cases are clear